source/guides/community-chat.rst

```diff
@@ -1,26 +1,44 @@
+Connect with thousands of contributors, customers, and users to build, share, and learn together at `community.mattermost.com <https://community.mattermost.com>`__. This server is our virtual office and is open to everyone. Please review our `Code of Conduct <https://handbook.mattermost.com/contributors/contributors/guidelines/contribution-guidelines>`__ before participating.
 
-We have many channels on our community server you can join. Here are some of the more popular channels.
+General channels
+----------------
 
-**General purpose channels**
+- `~Contributors <https://community.mattermost.com/core/chann0els/tickets>`__
+- `~Community Team <https://community.mattermost.com/core/channels/community-team>`__
+- `~Developers <https://community.mattermost.com/core/channels/developers>`__
+- `~New Channel Notifications <https://community.mattermost.com/core/channels/new-channel-notifications>`__
+- `~Off-Topic <https://community.mattermost.com/core/channels/off-topic-pub>`__
+- `~Reception <https://community.mattermost.com/core/channels/town-square>`__
 
-- `Peer-to-peer Help: <https://community.mattermost.com/core/channels/peer-to-peer-help>`__ Get help from other community members on using, deploying, or contributing to Mattermost.
-- `Ask Anything: <https://community.mattermost.com/core/channels/ask-anything>`__ Ask general questions about Mattermost.
+Technical support channels
+--------------------------
 
-**Contributor discussions**
+To familiarize yourself with our community support process, start by reading the `Get Help <https://docs.mattermost.com/guides/get-help.html>`__ documentation page.
 
-- `Developers: <https://community.mattermost.com/core/channels/developers>`__ Get help with development issues from our team and other developers in our community.
-- `Developers Meeting: <https://community.mattermost.com/core/channels/developers-meeting>`__ Join the weekly developers meeting to find out what Mattermost developers are working on.
-- `Contributors: <https://community.mattermost.com/core/channels/tickets>`__ Connect with other people who are working to make Mattermost better.
-- `Contributor Guilds: <https://developers.mattermost.com/contribute/getting-started/guilds/>`__ Find focused groups within our community that work on specific portions of the Mattermost ecosystem.
-- `Documentation Working Group: <https://community.mattermost.com/core/channels/dwg-documentation-working-group>`__ Provide feedback about Mattermost documentation and ask questions to the Mattermost Technical Writing team.
-- `Content: <https://community.mattermost.com/core/channels/mattermost-blog>`__ Discuss blog, video, podcast, and other content publication with the Mattermost Content team.
+- `~Ask Anything <https://community.mattermost.com/core/channels/ask-anything>`__
```

Comment:
What is the difference between 'ask anything' and 'peer-to-peer' help channels?

Reply:
Related question: Do we want to highlight that there's a channel where community members can test product functionality?

Reply:
@jasonblais Ahh, the biggest question I asked myself while making this change! It just looks like P2P Help has 2.5x the member count and is more focused on nitty-gritty troubleshooting, whereas Ask Anything has more "philosophical" questions about Mattermost.

@cwarnermm For brevity I'm not listing the [Public Test Channel](https://community.mattermost.com/core/channels/public-test-channel) here, because it doesn't seem to be heavily used anyways. But I'm open to adding it if we regularly refer members there. Just doesn't look like the case currently.

Reply:
Thanks @azigler - I suppose I didn't see a fundamental difference in the questions asked between the two channels, and the peer-to-peer help channel isn't overly busy (~100 messages a month it seems).